packages/grafana-ui/src/components/ClipboardButton/ClipboardButton.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { PureComponent } from 'react';
 import Clipboard from 'clipboard';
-import { Button, ButtonProps } from '../Button/Button';
+import { Button, ButtonProps } from '../Forms/Button';
```

Comment:
This change looks strange, should it not be that we move the old Button to /Forms/Legacy ?